src/advanced/bitcoin-anchoring.md

```diff
@@ -456,10 +452,10 @@ GET {base_path}/actual_lect/{id}
 ```
 
 Returns the actual LECT for the specified validator, along with the
-hash of Exonum transaction published this LECT.
+hash of the Exonum transaction published in this LECT.
 
-If the specified `id` is greater or equal to validators
-amount, returns an error.
+If the specified `id` is greater or equal to validator's
```

Comment:
> If the specified `id` is greater or equal to validator's value, returns an error.

The meaning is different:
"If the specified validator `id` is greater or equal to total validators amount, an error is returned."